builtin/credential/cert/backend.go

```diff
@@ -57,7 +69,32 @@ func (b *backend) invalidate(_ context.Context, key string) {
 		b.crlUpdateMutex.Lock()
 		defer b.crlUpdateMutex.Unlock()
 		b.crls = nil
+	case key == "config":
+		// Is this really necessary?
```

Comment:
The alternative to this lock would be an Atomic which is what we did in PKI.

https://github.com/hashicorp/vault/blob/main/builtin/logical/pki/crl_util.go#L72